### ISSUE TITLE
Changes required for 0.1.0 release

### ISSUE THREAD

**Issue by bruno-f-cruz** [AllenNeuralDynamics/aind-vr-foraging#87]
High priority (we will definitely add this)

- [x] Build one more box in medium size boxes. 
- [x] Any necessary changes to the task logic
- [x] Add olfactometer (ideally to all boxes)
- [x] Add photodiode (currently waiting for a mechanical design to hold it against the screen)
- [x] #129
- [x] Refactor logging folders 
- [x] Reimage all the computers
- [x] Lock versions in the schema and deployment
- [x] Thermistor board (The non-harpified board is ready could read it with Behavior-AnalogInput, any ETA on the harp version @Poofjunior ?)
- [x] #123
- [x] Add IR light and controller to all systems and all boxes
- [x] Change visual textures to black and white

Medium priority (we would like to have this)
- [x] Add new stage (waiting for the parts)
- [x] Reorganize cables and connections in white boxes. 
- [x] Replace metal headposts. 
- [x] Pogo pins systems on the wheel caddy. 
- [x] Add mirror to image animal face better
- [ ] Lock breathing to odor presentation (?)

Low priority (It would be nice, but it can wait until the next version or two)
- [x] Try to come up with a way to backup data daily (we could just wait for the SC team?)
- [x] #171

**Room infrastructure**
- [ ] Engineer new air pressure circuit. There should be 2 major valves for each of the 2 systems, then each box should have their own valve as well. 
- [ ] Add vacuum to the boxes

**Wont fix in this release**
- [ ] Lock breathing to odor presentation (?)
- [ ] Calibration of water + saving calibration values. 
- [ ] Try to align the rig and behaviorsession schemas (would be nice to wait for the new release of the sgen package)
- [ ] Motor/Wheel/Break controller @Poofjunior 
- [ ] Mouse database (waiting on SLIMS)
- [ ] Automatic training (perhaps still too soon for it, but would be good to start thinking about it)
